lib/galaxy/tools/imp_exp/__init__.py

```diff
@@ -122,7 +123,7 @@ def get_tag_str(tag, value):
                 datasets_usage_counts = {}
                 for dataset_attrs in datasets_attrs:
                     temp_dataset_file_name = \
-                        os.path.abspath(os.path.join(archive_dir, dataset_attrs['file_name']))
+                        os.path.realpath(os.path.abspath(os.path.join(archive_dir, dataset_attrs['file_name'])))
```

Comment:
You can remove the redundant call to `os.path.abspath()`, which is called internally by `os.path.realpath()` .